### PR TITLE
Bugfixes/slate upgrade

### PR DESCRIPTION
**Pull request by ivanaveliskova**

### CODE REVIEW

**Review by Jacfem — COMMENTED** [review]
Nice find!

Two thoughts/questions:
- Would you mind leaving a comment in the code of why we're commenting this out? Something like, historically, enter does not create a new row with the guru editor, so we don't want this new functionality without a larger conversation.
- Were you able to test this out by linking this to guru-slate-tools, and then linking that to Guru-UI? I can definitely run through this too to verify, because we should verify before merging to prevent any more changes needed.

**Comment by ivanaveliskova** [comment]
> Nice find!
> 
> Two thoughts/questions:
> 
> * Would you mind leaving a comment in the code of why we're commenting this out? Something like, historically, enter does not create a new row with the guru editor, so we don't want this new functionality without a larger conversation.
> * Were you able to test this out by linking this to guru-slate-tools, and then linking that to Guru-UI? I can definitely run through this too to verify, because we should verify before merging to prevent any more changes needed.

Yeah, I linked this branch to the guru-slatehtmlparser branch you have open and then I ran it locally by yarn linking guru-ui to that (to avoid having multiple yarn links to manage).

And yeah I forgot to delete the old code, so I can do that to clean this up, we can always refer back to the old commit if we ever want that functionality back.

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "watchify": "^3.7.0"
   },
   "scripts": {
-    "prepublish": "babel ./lib --out-dir ./dist",
+    "build": "babel ./lib --out-dir ./dist",
```
Comment (ivanaveliskova) [inline package.json:49]: I updated this just so we do a yarn build for the dist folder which is what we use. I didn't like using "prepublish"

Comment (YurkaninRyan) [reply package.json:49]: yeah this is actually an npm hook but since we don't have an npm repository it's better to switch the name

**Review by YurkaninRyan — APPROVED** [review]

**Pull request merged**